front/src/Phaser/Login/CustomizeScene.ts

```diff
@@ -40,7 +40,7 @@ export class CustomizeScene extends AbstractCharacterScene {
         this.layers = loadAllLayers(this.load);
         this.loadCustomSceneSelectCharacters().then((bodyResourceDescriptions) => {
             bodyResourceDescriptions.forEach((bodyResourceDescription) => {
-                if(!bodyResourceDescription.level){
+                if(bodyResourceDescription.level == undefined || bodyResourceDescription.level < 0 || bodyResourceDescription.level > 5 ){
```

Comment:
I think that level = -1 for the custom textures is applied for textures that are meant to be "global" characters.
Are you taking that into account?

Reply:
I check the code and in `AbstractCharacterScene` in function `loadCustomSceneSelectCharacters` apply a filter on the list of textures and by pass level -1.